README.md

```diff
@@ -25,7 +25,17 @@ to pass these lints. The recommended lints include all the core lints.
 
 Additionally, a third lint set -
 [`package:flutter_lints`](https://pub.dev/packages/flutter_lints) - extends the
-recommended set with additional recommended Flutter-specific lints. Documentation on that can be found at [github.com/flutter/packages](https://github.com/flutter/packages/tree/main/packages/flutter_lints).
+recommended set with additional recommended Flutter-specific lints.
+Documentation on that can be found at
+[github.com/flutter/packages](https://github.com/flutter/packages/tree/main/packages/flutter_lints).
```

Comment:
I'm not sure this adds any value over the earlier pub.dev link since the README contents are there as well and it links to the GitHub page. Consider dropping it.

```suggestion
```